Spokestack/TextToSpeech.swift

```diff
@@ -209,26 +287,26 @@ import CryptoKit
     }
 }
 
-fileprivate struct TTSSSMLResponseURL: Codable {
-    let url: URL
+public struct TTSSSMLResponseURL: Codable {
```

Comment:
Why do these need to be public? Is it for the publisher?

Reply:
I'm not sure why that was changed. that can be reverted

Reply:
Yes. was made `public` for the publisher. That will be reverted since the separate class will be deleted.